### ISSUE TITLE
Selected language in settings resets when changing pages

### ISSUE THREAD

**Issue by pushkar707** [lingua-libre/SignIt#56]
![Screenshot (21)](https://github.com/lingua-libre/SignIt/assets/77536989/7150df19-031a-4df3-9167-b51a56133ba0)

Issue: A lot of time the language set by the user for the extension gets reset, this behavior is seen usually when we open a new tab or just navigate through various URLs in the website.

Reproduction: Change the default language of the extension to any language (English in my case), and just use the browser normally, and keep checking the extension popup's language. You will be very quickly able to see the language change.

My System:
OS: Windows 11 with WSL (Ubuntu 20.04)

**Comment by pushkar707** [comment]
No, I ran the extension in Firefox browser, through "npm run web-ext:test" command

**Comment by pushkar707** [comment]
I think this was an issue on my end only, I will confirm it and close this.

**Comment by hugolpz** [comment]
@pushkar707 , I assigned you this task since you are leading it as well and (finger crossed 🤞🏼 ) it will self-solve.
